src/components/staking/PageValidator.vue

```diff
@@ -272,6 +250,11 @@ export default {
         `undelegations`,
         `transactions`
       ]) // TODO use more finegrained query string (network and address)
+    },
+    chooseFilter(field) {
+      return field
+        ? this.$options.filters.percent(field)
```

Comment:
In case of "Last Commission Change" this is the `fromNow` filter. How about passing in the filter as a second argument? Or can we chain the filters better @colw ?

Reply:
I would think `abc | percent | noBlanks` should work. 

`noBlanks` should just display as is, or `--` if empty.

Reply:
No because `| percent` transforms `null` into 0, and `0` is not a value `noBlanks` considers as "blank".